frontend/templates/views/detail/success-story-detail.j2

```diff
@@ -1,4 +1,6 @@
 {% extends '/layouts/default.j2' %}
+{% do doc.amp_dependencies.add('amp-vimeo', '0.1') %}
```

Comment:
Are always `amp-vimeo` and `amp-video` are used both? Otherwise they need to be inside the corresponding `{% if %}` clause.